docs/collections/documents/add-document.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Add Documents to Collection
+sidebar_position: 30
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page explains how to add a document to a Document Store collection.
+
+By default, a new document has only one attribute, `_key`. Defining the document `_key` attribute is optional. If you do not specify a key, then it will be autogenerated.
+
+## Add a Document with the Console
+
+Follow these instructions to add documents to an existing document collection using the GDN console web UI.
+
+1. Log in to your Macrometa account.
+1. Click **COLLECTIONS**.
+1. In the collection list, click the name of the document collection that you want to add a document to. If you aren't sure which collections are document collections, then you can click **Document** at the top of the page to see just document collections.
+1. Click **New Document**.
+1. Enter a `_key` if you want to and then click **Create**.
```

Comment:
```suggestion
1. (Optional) Enter a `_key`.
1. Click **Create**.
```

This one is probably not necessary, but I typically like to separate optional parts into an (Optional) step.  Your call either way.